src/console_cmds.cpp

```diff
@@ -9,6 +9,7 @@
 
 /** @file console_cmds.cpp Implementation of the console hooks. */
 
+#include <time.h>
```

Comment:
There's no hard-and-fast rule for this, but standard headers are generally put at the bottom of the list on their own (but before safeguards.h)